[documentation] Add a rocSOLVER tutorial (#510)
ROCm/hipfort: CHANGELOG.md

### Added

> * Added a *rocSOLVER examples* documentation page that walks through complete

Reviewer: ```suggestion
* Added a rocSOLVER tutorial in the documentation that walks through complete
```

  Fortran programs for LU factorization and solve (including the batched,
  strided-batched, and 64-bit APIs), Cholesky, QR and its `orgqr`/`ormqr`
  follow-ons, least squares, symmetric eigenvalues, the SVD, the symmetric
  indefinite factorization, the triangular inverse, and the reductions to
  condensed form. The page includes the test sources directly, so the documented

>   code is the code that is built and run by CTest.

Reviewer: ```suggestion
  condensed form.
```
